extensions/podman/src/extension.ts

```diff
@@ -879,15 +879,7 @@ export async function activate(extensionContext: extensionApi.ExtensionContext):
     });
   }
 
-  // no podman for now, skip
-  if (isMac()) {
-    if (!fs.existsSync(podmanMachineSocketsDirectory)) {
-      return;
-    }
```

Comment:
I think this check was added b/c there was an issue monitoring a non-existing directory on macOS so probably it'll fail ?

Reply:
So i think it must be moved somewhere else because in this way it returns and it does not start the monitorMachines at all.
If you just install podman without reactivating the extension it will never monitor the machines

Reply:
yes, I mean I understand your usecase but probably yes it needs to be moved vs being deleted 👍

Reply:
I tried on my Mac and it still works either if i delete the folder at runtime or if i remove it just before starting Desktop.

By looking the code i do not see where it could fail. All the functions executed after this point should not break by the missing folder. Both the monitorMachine/Provider have a silent catch which would prevent them to fail, then we just start the registry and proxy setup.

By going backwards i saw that the code was changed here https://github.com/containers/podman-desktop/commit/653b82bf4164fa5470b5caf6d3047034d911f59d#diff-c885162dd627b7cd812c83544b19b9b166179fec79ddbc13273fede7c2566791L238 
before it was watching that folder. Maybe this is what you recalled? In this case maybe it's not necessary anymore. 

Otherwise i need some guidance because i have no idea where i should move this check.

Reply:
I was remembering https://github.com/containers/podman-desktop/pull/188/files

so maybe it works with a non existing folder but not on a non-existing file

Reply:
also it seems it was Linux so we should be good on macOS then you tested it